sqlite3.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name     = 'sqlite3'
-  s.version  = '3.46.1'
+  s.version  = '3.46.2'
```

Comment:
The major.minor.patch triple must always be in sync with the SQLite version this pod is including. And since SQLite 3.46.2 doesn't exist yet, we can't use this. You can use `3.46.1+1` so that we can do another release for SQLite 3.46.1.

Reply:
Oh, I didn't know that. It's fixed now.